ds3-autogen-utils/src/main/java/com/spectralogic/ds3autogen/utils/ConverterUtil.java

```diff
@@ -16,14 +16,18 @@
 package com.spectralogic.ds3autogen.utils;
 
 import com.google.common.collect.ImmutableList;
-import com.spectralogic.ds3autogen.api.models.Classification;
-import com.spectralogic.ds3autogen.api.models.Ds3Request;
+import com.google.common.collect.ImmutableMap;
+import com.google.common.collect.ImmutableSet;
+import com.spectralogic.ds3autogen.api.models.*;
 
 import java.util.List;
 import java.util.Map;
+import java.util.Set;
 
 public class ConverterUtil {
 
+    private static final String CONTRACT_DEFINED_TYPE = "com.spectralogic.";
+
     public static boolean hasContent(final List<?> list) {
```

Comment:
Change this to `Collection` instead of `List`, and then you can pass `Set` classes into it.